MakeitMVP/CoffeeChat
# Style Search page and 3 tab components

## Summary
This code styles the Search page and its 3 tab components: Background, Mentorship, and Interests.

## Updates
Mockup: 
![image](https://github.com/MakeitMVP/CoffeeChat/assets/63386589/254d0e9d-0e1d-4f94-a6df-12d163611d7c)

Feature branch: 
<img width="485" alt="Screenshot 2023-11-14 at 9 35 56 PM" src="https://github.com/MakeitMVP/CoffeeChat/assets/63386589/9302d144-deb2-402f-b2cd-552508582f33">


## Testing Instructions
1. In the main branch, run `git pull` in your terminal to retrieve all updates, then switch to feature branch with `git checkout lc80-fe-styleSearchPage`
2. Start the dev environment with `npm start`
3. In your browser, navigate to the /search route to view the build of the hi-fi mockup. Click between the 3 tabs to see each component.
